src/problem1.rb

```diff
@@ -0,0 +1,17 @@
+class Problem1
+  def sumOfAllEvenNumbers(values)
+    response = 0
+    if(!values.nil? && (values.instance_of?(Array)))
+      if(values.all? {|i| i.is_a?(Integer)})
+        values.select {|i| i % 2 == 0}.each {|number|
```

Comment:
Is there a [better](http://ruby-doc.org/core-2.2.0/Integer.html#method-i-even-3F) way to express that a number is even in ruby?